main.py

```diff
@@ -1 +1,24 @@
 #Mad Libs Generator Project
+
+#Loop back to this point once code finishes
+loop = 1
+while (loop < 9):
```

Comment:
The game should loop so the user can create 10 stories.
So the while loop may be changed to loop <= 10, and can run 10 times for the stories.